docs/source/get-started/package-managers.rst

```diff
@@ -6,19 +6,100 @@ Use your favorite package manager to install Kokkos.
 System package managers
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-**DNF**
+DNF
+---
+
 You may use the Fedora Project package manager to install Kokkos
 https://packages.fedoraproject.org/pkgs/kokkos/
 
 Other package managers
 ~~~~~~~~~~~~~~~~~~~~~~
 
-`Spack <https://spack.io>`
---------------------------
+`Spack <https://spack.io>`_
+---------------------------
+
 Spack is a popular package manager for HPC.  Spack comes with installation recipes for Kokkos.
-https://packages.spack.io/package.html?name=kokkos
+
+The `Kokkos recipe webpage <https://packages.spack.io/package.html?name=kokkos>`_ summarizes the available versions of Kokkos
+and their options.
+
+Most of the time, Spack Kokkos' variants follow the same options as the Kokkos `CMake options <./configuration-guide.html>`_.
+List of available variants can be found by running
+
+.. code-block::
+
+    spack info kokkos
+
+
+Note that CPU micro-architectures are automatically inherited from Spack's configuration.
```

Comment:
I don't quite understand that sentence.
Does that mean Spack is auto-detecting your CPU arch and passing flags to Kokkos?

Reply:
Yes my understanding spack auto detects it to give a default value to `arch`. Then the Kokkos recipe does the mapping to the corresponding Kokkos `ARCH` option.

As @cedricchevalier19  said, if needed you can manually specify `arch`.